doc/source/api/core/solver/fielddata.rst

```diff
@@ -29,104 +28,118 @@ The field data object is an attribute of the solver object:
+  {4: array([-0.34724122,  0.        , -0.0442204 , -0.34724477, ...,
+         -0.04050309, -0.34645557,  0.        , -0.04421487, -0.34646705], dtype=float32)}
+       
+Response contains a dictionary of surface IDs to numpy array of the requested field. 
+
+get_scalar_field_data
+~~~~~~~~~~~~~~~~~~~~~
 You can call ``get_scalar_field_data`` to get scalar field data, such as temperature:
 
 .. code-block:: python
 
-  >>> field_data.get_scalar_field_data(surface_name="wall-elbow", field_name="temperature")
-  {0: array([0., 0., 0., ..., 0., 0., 0.], dtype=float32)}
+  >>> field_data.get_scalar_field_data(surface_name="inlet", field_name="temperature")
+  {4: array([922.6778 , 923.7954 , 923.791  , 922.67017, 922.88135,  ...,
+         924.94556, 924.9451 , 924.9536 , 923.87366, 922.8334 , 922.82434], dtype=float32)}
+         
+Response contains a dictionary of surface IDs to numpy array of the requested field.          
 
-
-You can call ``get_vector_field_data`` to get vector scalar field data, such as velocity:
+get_vector_field_data
+~~~~~~~~~~~~~~~~~~~~~
+You can call ``get_vector_field_data`` to get vector field data. 
 
 .. code-block:: python
 
-  >>> field_data.get_vector_field_data(surface_name="wall-elbow", vector_field="velocity")
-  {0: (array([0., 0., 0., ..., 0., 0., 0.], dtype=float32), 1.0)}
-
+  >>> field_data.get_vector_field_data(surface_name="inlet", vector_field="velocity")
+  {4: (array([ 5.81938386e-01, -1.01187916e+01,  3.14891455e-03,  ...,
+        -6.49216697e-02,  1.44923580e+00, -1.04387817e+01], dtype=float32), 0.00012235096)}
+  
+Response is a dictionary of surface IDs to a tuple containing a numpy array of ``vector field`` and ``vector-scale``. 
 
-The above example employs a separate method for requesting each type of field:
+get_pathlines_field_data
+~~~~~~~~~~~~~~~~~~~~~~~~  
+You can call ``get_pathlines_field_data`` to get pathlines data. 
 
-- ``get_surface_data`` for surface data.
-- ``get_scalar_field_data`` for scalar field data.
-- ``get_vector_field_data`` for vector field data.
+.. code-block:: python
 
-For a surface or scalar field request, the response contains a dictionary of surface IDs to numpy array of 
-the requested field. 
+  >>> field_data.get_pathlines_field_data(surface_name="inlet", field_name="temperature")
+  {4: {'faces': array([    2,     0,     1, ...,     2, 28828, 28829]), 'vertices': array([-0.34724122,  0.        , -0.0442204 , ..., -0.19877256,
+         -0.1250188 , -0.0621446 ], dtype=float32), 'temperature': array([879.25073, 828.8636 , 861.65686, ..., 892.1276 , 894.6389 ,
+         897.95325], dtype=float32)}}
 
-``surface_id [int] -> field[numpy.array]``
-  
-For a vector field request, the response is a dictionary of surface IDs to a tuple containing a
-numpy array of ``vector field`` and ``vector-scale``. 
+Pathlines data is returned as line surface. So response is a dictionary of surface IDs to a information about line surface. 
```

Comment:
We should be consistent in terminology. Is it simply "pathlines data" or "pathlines field data"?